### PR TITLE
[ENG-948] - Added keepalive http/s sockets

### PR DESCRIPTION
**Pull request by Phara0h**
| JIRA  | [ENG-948](https://abedev.atlassian.net/browse/ENG-948)
| ----- | -

## Description
Added keepalive http/s sockets

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,10 @@
   "dependencies": {
     "eslint": "5.16.0",
     "eslint-plugin-security": "1.4.0",
+    "hashids": "^1.2.2",
     "pg": "7.11.0",
     "pg-native": "3.0.0",
-    "uuid": "3.3.2"
+    "uuid": "3.3.2",
+    "zeromq": "^5.1.0"
```
Comment (tmak) [inline package.json]: `hashids` and `zeromq` don't seem to be part of this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.1.2 - 07/1/2019
```
Comment (tmak) [inline CHANGELOG.md]: This sounds more like a `1.2.0` to me than a bugfix release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+
+-   Added keepalive http/s sockets [ENG-873](https://abedev.atlassian.net/browse/ENG-948)
```
Comment (rstonebraker-abe) [inline CHANGELOG.md]: readme mismatch

File: package.json (hunk 1 above)
Comment (Phara0h) [reply package.json]: Ahh yea ill remove those

File: CHANGELOG.md (hunk 2 above)
Comment (Phara0h) [reply CHANGELOG.md]: You think so? Alrighty ill change it to 1.2

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## 1.1.2 - 07/1/2019
```
Comment (tmak) [inline CHANGELOG.md]: This version is still wrong

File: lib/request.js [hunk 5]
```diff
@@ -2,15 +2,30 @@ var client = {
     https: require('https'),
     http: require('http'),
 };
+var agent = {
+    http: new client.http.Agent(
+        {
+            keepAlive: true
```
Comment (kody-abe) [inline lib/request.js]: Should we be able to specify `keepAliveMsecs`, `maxFreeSockets`, and/or `timeout` as well?

Comment (Phara0h) [reply lib/request.js]: I like the default values.

Comment (kody-abe) [reply lib/request.js]: Sure, we can still keep the defaults - But the ability to override them might be helpful in the future if needed. No?

Comment (Phara0h) [reply lib/request.js]: maxFreeSockets = 256
keepAliveMsecs = 1000
timeout = 40000 (if it is the same as server, does not mention it in api docs) 

all things we really don't want to mess with honestly.

**Pull request merged**